packages/discovery-provider/src/queries/get_attestation.py

```diff
@@ -157,6 +161,17 @@ def get_attestation(
     if disbursement:
         raise AttestationError(ALREADY_DISBURSED)
 
+    if challenge.weekly_pool:
+        now = datetime.now(pytz.timezone("US/Pacific"))
```

Comment:
Is there any particular reason we're storing and operating specifically in PST? I would think using UTC and calculating the appropriate matching time (4pm UTC) would be less error prone. I may be overcautious about time offset errors in places where we expect a certain timezone and get another.

Reply:
I'll switch to UTC - it's a good point. I was preferring PT because it may be easier on the reader to rationalize and I don't think we would have issues here because operationally everyone should have the same view of pytz US/Pacific